fakeserver/testdata/.gitignore

```diff
@@ -0,0 +1 @@
+assignments.yml
```

Comment:
I'm wondering if it would be better to gitignore this whole directory and then mkdir/rm in tests. That way if we every persist any other files, we won't have to update existing tests to clear those out too.

Reply:
(related: just commented elsewhere about using `ioutil.TempDir` for test data)